tests/integration/backend_test.py

```diff
@@ -423,43 +423,6 @@ def test_classical(
     assert backend.run_circuit(c, n_shots=10, language=language).get_counts()  # type: ignore
 
 
-@pytest.mark.skipif(skip_remote_tests, reason=REASON)
-@pytest.mark.parametrize(
-    "authenticated_quum_backend",
-    [{"device_name": name} for name in pytest.ALL_SYNTAX_CHECKER_NAMES],  # type: ignore
-    indirect=True,
-)
-@pytest.mark.parametrize(
-    "language",
-    [
-        Language.QASM,
-        pytest.param(
-            Language.QIR,
-            marks=pytest.mark.xfail(
-                reason="https://github.com/CQCL/pytket-quantinuum/issues/299"
-            ),
-        ),
-    ],
-)
-@pytest.mark.timeout(120)
-def test_division(
```

Comment:
Rather than delete these tests we could modify them so that they expect an exception to be raised; then swap the "xfail" annotation between QASM and QIR.

(I think we will want to support both these cases one day.)

Reply:
Good point, done in the added commits.